### PR TITLE
Threaded comparisons

### PR DESCRIPTION
**Pull request by pal03377**
This PR makes use of processes to parallelize the ML code comparisons. I tried using threads first, but that wasn't faster at all.

Using 10 test submissions including 27 feedbacks ([test-submissions.zip](https://github.com/ls1intum/Themis-ML/files/10541549/test-submissions.zip)), the server took ~57s before on my laptop.

Now, using multiple processes to parallelize it, it takes ~25s.

This is not the amazing improvement I hoped for, but it's still around 2x, which is nice.

Also, I had to fix the `/inject` prefix by also adding the root endpoint to it and instead using `http://localhost:8001/inject` as the server URL, which worked nicely. I was the reviewer on the PR that broke that, so sorry for not noticing 😬  Now it works again (if you choose the server URL like in the updated documentation).

I renamed the `feedback_text` column to `text`, so you might need to adapt / re-setup your DB as well.

Edit: More nice stats: for 216 feedbacks (the 27 from before duplicated 8 times), it takes 7m13s using the old approach. The new one took 1m23s 🎉  Apparently the setup costs for the processes are pretty high.

### CODE REVIEW

**Comment by huber-florian** [comment]
Sounds great!

File: feedback-suggestion/src/app/endpoints/feedback_suggestion_request.py [hunk 1]
```diff
@@ -32,6 +34,11 @@ def load_feedbacks(request: NotifyRequest):
 
     response = auth_request.get(
         f"/programming-exercise-participations/{request.participation_id}/latest-result-with-feedbacks")
+    if response.status_code == 404:
+        raise HTTPException(status_code=404,
+                            detail="No result found for assessment. Assessment might not be submitted.")
+    if response.status_code != 200:
+        raise HTTPException(status_code=response.status_code, detail="Error on Artemis server: " + response.text)
```
Comment (botrufus) [inline feedback-suggestion/src/app/endpoints/feedback_suggestion_request.py:41]: Would put this it into its own function

**Review by botrufus — APPROVED** [review]
Took a relatively quick look, but looks good!

File: feedback-suggestion/src/app/endpoints/feedback_suggestion_request.py (hunk 1 above)
Comment (pal03377) [reply feedback-suggestion/src/app/endpoints/feedback_suggestion_request.py:41]: did that in #12 👍

**Pull request merged**